Dockerfile

```diff
@@ -73,5 +72,7 @@ RUN \
     openssl-devel mariadb-devel sendmail curl wget help2man perl-libwww-perl && \
     yum clean all && \
     rm -rf /var/cache/yum/* && \
+    sed -i '/session    required   pam_loginuid.so/c\#session    required   pam_loginuid.so' /etc/pam.d/crond && \
+    chmod 0644 /etc/crontab && \
```

Comment:
Nice catch.  The permissions on `/etc/crontab` were preventing the poller from running